vue-frontend/src/components/jobs/PerksAndBenifits.vue

```diff
@@ -253,6 +264,7 @@ export default {
   }
 }
 
+
```

Comment:
Configure .githooks for beautify code.
The command is given in readme.md, review it